scripts/package.json

```diff
@@ -16,7 +16,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@dcl/protocol": "^1.0.0-5125382977.commit-f7db98f",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-8363421289.commit-3864902.tgz",
```

Comment:
I'm not 100% sure yet, but I don't think we should commit this? Maybe I'm wrong and I'd appreciate any insights :)

Reply:
The protocol itself is not 100% ready so I leave it be for now